src/commands/utils.js

```diff
@@ -0,0 +1,16 @@
+export function getI18nKindWarning(
+  warnings,
+  constants,
+  getToken = (i18nToken) => i18nToken
+) {
+  return warnings.map((warning) => {
+    return {
+      ...warning,
+      kind: getToken(
```

Comment:
Just a question, why `kind` and not `value` or `local` ?

Reply:
Because historically the type of `warning` object is `BaseWarning`

```
// package js-x-ray
// index.d.ts

interface BaseWarning {
    kind: "unsafe-import" | kindWithValue;
    file?: string;
    value: string;
    location: WarningLocation | WarningLocation[];
  }
```